packages/chart/src/__tests__/heat.ts

```diff
@@ -28,7 +28,8 @@ export class Test extends Heat {
 
 //  Calculate the standard deviation of the weight column  ---
 const extractWeight = df.map(row => row[2]);
-const distribution = df.scalar(df.distribution())(extractWeight(data));
+export const distribution = df.scalar(df.distribution())(extractWeight(data));
+export const quartiles = df.scalar(df.quartile())(extractWeight(data));
```

Comment:
This "quartiles" const doesn't appear to be used later on in the test.

Reply:
Its exported and used in the StatChart test